test.sh.stderr

```diff
@@ -0,0 +1,22 @@
++ truncate -s 0 dummy.img
++ truncate -s 20GiB dummy.img
++ sfdisk --force dummy.img
++ sfdisk --dump dummy.img
+++ sudo losetup --nooverlap --find --sector-size 4096 --show -- dummy.img
++ loopdev=/dev/loop0
++ trap 'sudo losetup -D -- "$loopdev"' EXIT
++ make gptfixer/gpt
++ sudo sfdisk --dump -- /dev/loop0
+GPT PMBR size mismatch (41943039 != 5242879) will be corrected by write.
++ sudo gptfixer/gpt fix /dev/loop0
+sudo: gptfixer/gpt: command not found
```

Comment:
This can't possibly be the expected output...

Reply:
yeah, that was a (rather serious) bug.